src/platform/packages/shared/dashboards/README.md

```diff
@@ -0,0 +1,48 @@
+# `@kbn/dashboards`
```

Comment:
Can we rename this to something more specific? Maybe something like `@kbn/dashboards-selector`?

Reply:
Thanks, I had a hard time finding a good name 😂